core/src/main/java/com/amplifyframework/auth/result/AuthSocialSignInResult.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.auth.result;
+
+import androidx.core.util.ObjectsCompat;
+
+public class AuthSocialSignInResult {
+    private final boolean isSignedIn;
+
+    /**
+     * Wraps the result of a sign in operation.
+     * @param isSignedIn true if the user is now completely signed in, false otherwise
+     */
+    public AuthSocialSignInResult(boolean isSignedIn) {
+        this.isSignedIn = isSignedIn;
+    }
+
+    /**
+     * True if the user is now completely signed in, false otherwise.
```

Comment:
In the JavaDoc, can you qualify what "completely" means, here? Does it mean successful completion of any sign-in challenges, and (something about that validation data object?)

Reply:
Actually in thinking about it, I decided to remove the completely term since I could imagine a scenario where a user is signed in but there are still steps suggested to be taken so isSignedIn here should just be simply whether the user is actually authenticated. Also updated the comment to the technical term of authenticated to be precise.